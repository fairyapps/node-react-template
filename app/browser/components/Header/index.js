import React, { Component } from "react";
import { Container, Row, Column } from "../Grid";

class Header extends Component {
  render() {
    return (
      <header>
        <Container marginTop="20px" />
      </header>
    );
  }
}

export default Header;
