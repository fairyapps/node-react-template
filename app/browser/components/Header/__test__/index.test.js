import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Header from "../index";

test("Header renders correct snapshot", () => {
  const component = renderer.create(<Header>sometext</Header>);
  let tree = component.toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});
