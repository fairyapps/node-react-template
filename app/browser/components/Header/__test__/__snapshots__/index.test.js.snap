// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correct snapshot 1`] = `
.fYdnBr {
  max-width: 100%;
  margin: 0 auto;
  padding-right: 10px;
  padding-left: 10px;
  margin-top: 20px;
}

@media (min-width: 768px) {
  .fYdnBr {
    max-width: 750px;
  }
}

@media (min-width: 992px) {
  .fYdnBr {
    max-width: 970px;
  }
}

@media (min-width: 1200px) {
  .fYdnBr {
    max-width: 1170px;
  }
}

<header>
  <div
    className="sc-bdVaJa fYdnBr"
  />
</header>
`;
